udm/intermediate/shopify-samples/orders-json/Backorder BOPIS - content.json

```diff
@@ -0,0 +1,391 @@
+{
+  "order": {
+    "id": 5524199899389,
+    "admin_graphql_api_id": "gid://shopify/Order/5524199899389",
+    "app_id": 580111,
+    "browser_ip": "111.118.250.178",
+    "buyer_accepts_marketing": false,
+    "cancel_reason": null,
+    "cancelled_at": null,
+    "cart_token": "c1-205acf291edff427599b826e4816ad86",
+    "checkout_id": 34877545906429,
+    "checkout_token": "3a118dc38d9f924f60fdb5e7350f9923",
+    "client_details": {
+      "accept_language": "en-IN",
+      "browser_height": null,
+      "browser_ip": "111.118.250.178",
+      "browser_width": null,
+      "session_hash": null,
+      "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
+    },
+    "closed_at": null,
+    "company": null,
+    "confirmed": true,
+    "contact_email": "muskan@hotwax.com",
+    "created_at": "2024-02-12T05:57:02-05:00",
+    "currency": "USD",
+    "current_subtotal_price": "69.00",
+    "current_subtotal_price_set": {
+      "shop_money": {
+        "amount": "69.00",
+        "currency_code": "USD"
+      },
+      "presentment_money": {
+        "amount": "69.00",
+        "currency_code": "USD"
+      }
+    },
+    "current_total_additional_fees_set": null,
+    "current_total_discounts": "0.00",
+    "current_total_discounts_set": {
+      "shop_money": {
+        "amount": "0.00",
+        "currency_code": "USD"
+      },
+      "presentment_money": {
+        "amount": "0.00",
+        "currency_code": "USD"
+      }
+    },
+    "current_total_duties_set": null,
+    "current_total_price": "69.00",
+    "current_total_price_set": {
+      "shop_money": {
+        "amount": "69.00",
+        "currency_code": "USD"
+      },
+      "presentment_money": {
+        "amount": "69.00",
+        "currency_code": "USD"
+      }
+    },
+    "current_total_tax": "0.00",
+    "current_total_tax_set": {
+      "shop_money": {
+        "amount": "0.00",
+        "currency_code": "USD"
+      },
+      "presentment_money": {
+        "amount": "0.00",
+        "currency_code": "USD"
+      }
+    },
+    "customer_locale": "en-IN",
+    "device_id": null,
+    "discount_codes": [],
+    "email": "muskan@hotwax.com",
+    "estimated_taxes": false,
+    "financial_status": "authorized",
+    "fulfillment_status": null,
+    "landing_site": "/",
+    "landing_site_ref": null,
+    "location_id": null,
+    "merchant_of_record_app_id": null,
+    "name": "#2005",
+    "note": null,
+    "note_attributes": [],
+    "number": 1005,
+    "order_number": 2005,
+    "order_status_url": "https://hc-demo.myshopify.com/6973849727/orders/6eac8a69f05091e0660740c6c9ca8f64/authenticate?key=07830f972e5cb4106fb1dc8cd7e0260a",
```

Comment:
remove key and use hc-sandbox url

Reply:
@sanjanashrivastava Removed token values from all the JSON